app/controllers/buyers/applications/bulk/change_plans_controller.rb

```diff
@@ -10,11 +10,13 @@ def create
     # TODO: really change plan
     return unless (plan = service.application_plans.find_by(id: plan_id_param))
 
-    applications.each do |application|
-      @errors << application unless application.change_plan(plan)
-    end
+    applications.reject { |app| app.plan.id == plan.id }
```

Comment:
```suggestion
    applications.reject { |app| app.plan_id == plan.id }
```

We don't need to instantiate the Plan (it causes a `SELECT *` for the plan)